### ISSUE TITLE
3.5.21 update breaks unix socket

### ISSUE THREAD

**Issue by angmarb** [walkor/Workerman#455]
After updating 3.5.20 -> 3.5.21 
**unix socket** worker setting up 'transport' property as **tcp**. 
so all unix-socket actions inside Worker.php not working properly 
(like removing .sock file on shutdown)
example:
**test worker:**
```
root@5b6b25a24714:/usr/bin/worker# cat test.php 
<?php
include __DIR__.'/vendor/autoload.php';

$worker = new \Workerman\Worker('unix://'.__DIR__.'/tmp.sock');

\Workerman\Worker::runAll();
```
**now it is unix**
```
root@5b6b25a24714:/usr/bin/worker# php test.php start
Workerman[test.php] start in DEBUG mode
----------------------------------------------- WORKERMAN -----------------------------------------------
Workerman version:3.5.20          PHP version:7.2.19-0ubuntu0.18.04.2
------------------------------------------------ WORKERS ------------------------------------------------
proto   user            worker          listen                             processes    status           
unix    root            none            unix:///usr/bin/worker/tmp.sock    1             [OK]            
---------------------------------------------------------------------------------------------------------
Press Ctrl+C to stop. Start success.
^CWorkerman[test.php] stopping ...
Workerman[test.php] has been stopped
```
**updating to 3.5.21**
```
root@5b6b25a24714:/usr/bin/worker# composer require workerman/workerman 3.5.21
Do not run Composer as root/super user! See https://getcomposer.org/root for details
./composer.json has been updated                                                    
Loading composer repositories with package information
Updating dependencies (including require-dev)             
Package operations: 0 installs, 1 update, 0 removals
  - Updating workerman/workerman (v3.5.20 => v3.5.21): Loading from cache
Writing lock file
Generating autoload files
```
**and now it is tcp**
```
root@5b6b25a24714:/usr/bin/worker# php test.php start
Workerman[test.php] start in DEBUG mode
----------------------------------------------- WORKERMAN -----------------------------------------------
Workerman version:3.5.21          PHP version:7.2.19-0ubuntu0.18.04.2
------------------------------------------------ WORKERS ------------------------------------------------
proto   user            worker          listen                             processes    status           
tcp     root            none            unix:///usr/bin/worker/tmp.sock    1             [OK]            
---------------------------------------------------------------------------------------------------------
Press Ctrl+C to stop. Start success.
^CWorkerman[test.php] stopping ...
Workerman[test.php] has been stopped
```

**Comment by walkor** [comment]
Thank you for your post.
It fixed in version 3.5.22.

**Comment by angmarb** [comment]
The new version fixed it. Thanks.
